### PR TITLE
test: dappeteer

### PR DESCRIPTION
**Pull request by simboonlong**
#### updates
- test transfer and assert ebls on local blockchain
- related to https://github.com/TradeTrust/tradetrust-website/pull/210

#### remarks
- local tests worked `npm run integration:dappeteer`
- ci tests failed, need to get puppeteer to run in headful mode somehow (testing with metamask extension)

### CODE REVIEW

File: integration/local/index.mjs [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import puppeteer from "puppeteer";
+import dappeteer from "dappeteer";
+import { transferHolder } from "./transfer-holder.mjs";
+import { surrender } from "./surrender.mjs";
+
+const main = async () => {
+  const browser = await dappeteer.launch(puppeteer, {
+    headless: false, // https://github.com/puppeteer/puppeteer#default-runtime-settings
```
Comment (justussoh) [inline integration/local/index.mjs:8]: Why is it being run non-headless?

Comment (simboonlong) [reply integration/local/index.mjs:8]: it got stuck if headless:

Screenshot 2020-10-20 at 9 43 54 AM

https://github.com/TradeTrust/tradetrust-website/pull/253/checks?check_run_id=1278499255

im guessing this: 
- https://stackoverflow.com/questions/45372066/is-it-possible-to-run-google-chrome-in-headless-mode-with-extensions#:~:text=No%2C%20google%2Dchrome%20in%20headless,doesn't%20supports%20extensions%20yet.
- https://github.com/puppeteer/puppeteer/issues/659

File: scripts/integration-contracts.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
+account_key=0xe82294532bcfcd8e0763ee5cef194f36f00396be59b94fb418f5f8d83140d9a7
```
Comment (justussoh) [inline scripts/integration-contracts.sh:1]: Does this associate with any account? Should it be a secret?

Comment (simboonlong) [reply scripts/integration-contracts.sh:1]: this is actually the first key from the first 10 local blockchain accounts, when u use a fixed mnemonic to seed the blockchain. its deterministic as long as u used a fixed phrase + it's from local accounts so i think should be fine

**Review by justussoh — APPROVED** [review]
lgtm

**Pull request merged**